people_tool/static/people_tool/css/main.css

```diff
@@ -5,6 +5,12 @@
     pointer-events: none;
 }
 
+
+table.person-info-table {
```

Comment:
I moved the styles from the table to the css element. Didn't attempt using col-offset , but can lookup the search table.